plugins/modules/clusters.py

```diff
@@ -17,34 +17,44 @@
 description: Interact with clusters through the AssistedInstall API
 
 options:
-    action:
-      description: The action to perform (list, create, delete)
-      required: false
-      default: "list"
-      type: str
+    state:
+        description: The state to perform (present, absent, none for list)
+        required: false
+        choices: ["absent", "present"]
+        default: null
+        type: str
     cluster_id:
         description: The cluster ID to perform the action on
```

Comment:
```suggestion
        description: The cluster ID to perform the action on (required for cluster specific operations)
```

Reply:
^ This is not required for create operations, so for now I will not commit this one

Reply:
should it be reworded to `required for operations on a specific cluster`